SU-HSDO/suhumsci
# shs-132: adds mobile background on active pages

## Summary
This adds a background color to the mobile menu for the new megamenu on active pages. The reason is to align more closely with the old menu.

## Steps to Test
1. Go to a local site with the new megamenu in mobile format. Visited a nested page `/events/example-lecture-series`. Ensure the active link has a gray background.
2. Go one level up to `/default-views/event-series`. Ensure it has a background color and the expanded children do not.

## Screenshots
![Screen Shot 2023-01-11 at 10 57 37 AM](https://user-images.githubusercontent.com/8405274/211854050-59c3ee6a-a9b7-4638-a03d-7a970f86635e.png)
